### DISCUSSION TITLE
Get PR checks list from API/CLI

### DISCUSSION THREAD

**Discussion by aleks-ivanov** [community/community#24750] [Apps, API and Webhooks]
<p>Going through the responses from the /pulls endpoint, I am wondering is there a way to get through it a list of all the checks for a certain PR ? Or may be through the git/gh CLIs ?</p>
<p>To make it clear, I am trying to get something like this:</p>
<pre><code class="lang-auto">[
  checks: {
    'RNW Website PR / Website PR (pull_request)',
    'license/cla' 
  }
]
</code></pre>
<p>from this:</p>
<p><img src="https://user-images.githubusercontent.com/74899441/181086571-118d0850-1a7c-4992-ad2c-ea6ab803a6be.png" alt="24g24h353" data-base62-sha1="ku0MzHgQdRfAZ12gGNQXsw5Zl3" width="638" height="203"></p>
<p>The additional information like pass/fail, source etc. is also welcome :slight_smile:</p>

**Comment by parithiban** [comment, marked as answer]
<p><a class="mention" href="/aleks-ivanov">@aleks-ivanov</a></p>
<p>You can use GItHub <a href="https://github.com/cli/cli" rel="noopener nofollow ugc">CLI</a> to get the pull request checks.</p>
<p>Ex</p>
<ol>
<li>Success Report<br>
I have used <a href="https://github.com/microsoft/react-native-windows-samples" rel="noopener nofollow ugc">microsoft/react-native-windows-samples</a> PR <a href="https://github.com/microsoft/react-native-windows-samples/pull/450" rel="noopener nofollow ugc">450</a>.</li>
</ol>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/c/f/cfd93270c655e9ffa5176162d2f86d73512b5ac0.png" data-download-href="/uploads/short-url/tEIb5CgY0AH8PA5tODNlPbP0eTC.png?dl=1" title="Screenshot 2021-05-30 at 9.00.10 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/7281166/181086573-3ab1c41d-7a0a-4e66-8382-d88f40640cc9.png" alt="Screenshot 2021-05-30 at 9.00.10 PM" data-base62-sha1="tEIb5CgY0AH8PA5tODNlPbP0eTC" width="690" height="89" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/c/f/cfd93270c655e9ffa5176162d2f86d73512b5ac0_2_690x89.png, https://aws1.discourse-cdn.com/github/optimized/3X/c/f/cfd93270c655e9ffa5176162d2f86d73512b5ac0_2_1035x133.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/c/f/cfd93270c655e9ffa5176162d2f86d73512b5ac0_2_1380x178.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/c/f/cfd93270c655e9ffa5176162d2f86d73512b5ac0_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot 2021-05-30 at 9.00.10 PM</span><span class="informations">1972×256 45.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<ol start="2">
<li>Failure Report<br>
I have used <a href="https://github.com/microsoft/react-native-windows-samples" rel="noopener nofollow ugc">microsoft/react-native-windows-samples</a> PR <a href="https://github.com/microsoft/react-native-windows-samples/pull/446" rel="noopener nofollow ugc">446</a>.</li>
</ol>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/7/1758083792ae9ff68cb651b30848fd3bc04b575d.png" data-download-href="/uploads/short-url/3kvAySSF0myJN1HRqUqpDYISmG9.png?dl=1" title="Screenshot 2021-05-30 at 9.00.32 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/7281166/181086574-5f4fcd48-28c2-4206-bcea-031dbe0794e6.png" alt="Screenshot 2021-05-30 at 9.00.32 PM" data-base62-sha1="3kvAySSF0myJN1HRqUqpDYISmG9" width="690" height="188" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/7/1758083792ae9ff68cb651b30848fd3bc04b575d_2_690x188.png, https://aws1.discourse-cdn.com/github/optimized/3X/1/7/1758083792ae9ff68cb651b30848fd3bc04b575d_2_1035x282.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/1/7/1758083792ae9ff68cb651b30848fd3bc04b575d_2_1380x376.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/7/1758083792ae9ff68cb651b30848fd3bc04b575d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot 2021-05-30 at 9.00.32 PM</span><span class="informations">2116×578 129 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
